VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxParsePushBroadcastReceiver.java

```diff
@@ -32,101 +33,129 @@
     //UI and animation time need to get from production team.
     private boolean enableFlyinWindow = false;
```

Comment:
As a constant, this should have the `static final` modifiers to enable compile-time optimizations.